src/mappings/robo/RoboVaultV3DAIVaultMappings.ts

```diff
@@ -14,37 +15,57 @@ import {
   Withdraw2Call,
   Withdraw3Call,
   UpdatePerformanceFee as UpdatePerformanceFeeEvent,
-  StrategyAdded1 as StrategyAddedV2Event,
   UpdateManagementFee as UpdateManagementFeeEvent,
+  StrategyAdded1 as StrategyAddedV2Event,
   StrategyAddedToQueue as StrategyAddedToQueueEvent,
   StrategyRemovedFromQueue as StrategyRemovedFromQueueEvent,
   UpdateRewards as UpdateRewardsEvent,
-} from '../../../generated/ftmYvDAIVault/Vault';
+} from '../../../generated/RoboVaultV3DAI/Vault';
+
+/**
+ * This vault name should match import path above
+ */
+const VAULT_NAME = 'RoboVaultV3DAI';
+
+/**
+ * This mapping file will be superseded by the Registry depending on the block number below.
+ * All events after this block number will be indexed via the the generic mapping files instead (see vaultMapping.ts)
+ */
+const VAULT_ADDED_TO_REGISTRY = BigInt.fromI32(32166667); // https://ftmscan.com/tx/0xc8a5a9299b9b6231a2eddde8b8cddc4c5cdd16df7f7e30eeb8db0d1134d107c3
```

Comment:
This is an example of the hardcoded block number of when this vault was added to the registry. This mapping file isn't used after the Subgraph has indexed passed this event.